stdlib/public/runtime/SwiftObject.mm

```diff
@@ -34,7 +34,9 @@
 #include "../SwiftShims/RuntimeShims.h"
 #include "Private.h"
 #include "swift/Runtime/Debug.h"
+#if !defined(_MSC_VER)
```

Comment:
The uses of `dlsym` are guarded by `#if SWIFT_OBJC_INTEROP`, so let's use the same condition here as well.